src/vs/editor/contrib/clipboard/clipboard.ts

```diff
@@ -94,6 +94,9 @@ export const CopyAction = supportsCopy ? registerCommand(new MultiCommand({
 	}]
 })) : undefined;
 
+MenuRegistry.appendMenuItem(MenuId.MenubarEditMenu, { submenu: MenuId.EditorContextCopy, title: { value: nls.localize('copy as', "Copy As..."), original: 'Copy As...', }, group: '2_ccp', order: 3 });
+MenuRegistry.appendMenuItem(MenuId.EditorContext, { submenu: MenuId.EditorContextCopy, title: { value: nls.localize('copy as', "Copy As..."), original: 'Copy As...', }, group: CLIPBOARD_CONTEXT_MENU_GROUP, order: 3 });
+
```

Comment:
So the `EditorContextCopy` sub menu is shown in editor and the global edit menu? What about the explorer? Also, should there be a copy menu per container? Like `MenubarEditMenuCopy` etc?

Reply:
> What about the explorer?

Hmm it's really only mean for editor content. The titlebar CCP menu items don't work with explorer items.

> Also, should there be a copy menu per container? Like `MenubarEditMenuCopy` etc?

I don't mind, just that probably every adopter will have to contribute to both.

Reply:
@jrieken Any preference here?

Reply:
So, right now this is for the editor context menu and the global edit menu? Is the latter always scoped to the editor? When focus is in the explorer will cut, copy, and paste target files?

Reply:
Correct. The latter is always scoped to the editor. If I select a file in the Explorer, the menu Copy Paste actions don't do nothing.

Reply:
Then is makes sense. Given we wanna keep it this way, for undo/redo we changed the commands to be dynamic but that might not change a thing for the API itself. Then we can add a new copy-as menu and say the existing one only shows in editors